ui/src/main/resources/Meeting/Code/Migrations/2.0.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.2" reference="Meeting.Code.Migrations.2\.0" locale="">
+  <web>Meeting.Code.Migrations</web>
+  <name>2.0</name>
+  <language/>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1544623469000</creationDate>
+  <parent>Meeting.Code.Migrations.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1545214225000</date>
+  <contentUpdateDate>1545214225000</contentUpdateDate>
+  <version>1.1</version>
+  <title>2.0</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
+  <content>{{velocity}}
+#macro(handleWiki $w $meetingClass)
+  #set($dateFormat = "dd/MM/yyyy HH:mm")
+  #set($meetingClassDoc = $xwiki.getDocument("${w}:${meetingClass}"))
+  #if ($meetingClassDoc) ##&amp;&amp; !$meetingClassDoc.isNew())
+    #set($allMeetings = $services.query.xwql("from doc.object(${meetingClass}) as meeting order by doc.fullName asc").setWiki($w).addFilter('unique').execute())
+    | $w |$allMeetings.size()|(((
```

Comment:
It would be good to match the number of the table header's cells with the number of the table rows' cell (2 cells in header and 6 cells in other rows). Or at least we can make it easier to understand what the messages from the rows are meaning.